atrium/trcdb/db.go

```diff
@@ -203,13 +199,10 @@ func QueryWithBindings(te *engine.TierceronEngine, query string, bindings map[st
 	ctx.WithQuery(query)
 	queryLock.Lock()
 	//	te.Context = ctx
-	schema, r, queryErr := te.Engine.QueryWithBindings(ctx, query, bindings)
+	schema, r, err := te.Engine.QueryWithBindings(ctx, query, bindings)
 	queryLock.Unlock()
-	if queryErr != nil {
-		if strings.Contains(queryErr.Error(), "duplicate") {
```

Comment:
Are you sure this still shouldn't be included?

Reply:
Sure